pkg/proof/proof_test.go

```diff
@@ -1,66 +1,65 @@
-package proof
+package proof_test
 
 import (
 	"bytes"
-	"sort"
 	"testing"
 
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/celestiaorg/celestia-app/test/util/blobfactory"
 	"github.com/celestiaorg/celestia-app/test/util/testfactory"
 
 	"github.com/celestiaorg/celestia-app/pkg/da"
+	"github.com/celestiaorg/celestia-app/pkg/proof"
+	"github.com/celestiaorg/celestia-app/pkg/square"
 
 	"github.com/celestiaorg/celestia-app/pkg/appconsts"
 	appns "github.com/celestiaorg/celestia-app/pkg/namespace"
 	"github.com/celestiaorg/celestia-app/pkg/shares"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/tendermint/tendermint/types"
 )
 
 func TestNewTxInclusionProof(t *testing.T) {
-	blockData := types.Data{
-		Txs:        testfactory.GenerateRandomTxs(50, 500),
-		Blobs:      []types.Blob{},
-		SquareSize: appconsts.DefaultMaxSquareSize,
-	}
+	blockTxs := testfactory.GenerateRandomTxs(50, 500).ToSliceOfBytes()
```

Comment:
Will these contain transactions that have multiple blobs? If not, I guess we should test that also